MapboxSearch/place-autocomplete/src/main/java/com/mapbox/search/autocomplete/PlaceAutocompleteImpl.kt

```diff
@@ -97,19 +97,12 @@ internal class PlaceAutocompleteImpl(
         }
     }
 
-    private suspend fun createSuggestions(
+    private fun createSuggestions(
         rawSuggestions: List<BaseSearchSuggestion>
     ): Expected<Exception, List<PlaceAutocompleteSuggestion>> {
-        // In some cases searchEngine.resolveAll() can be significantly faster than suggestions resolving one by one,
-        if (rawSuggestions.all { it.coordinate == null }) {
-            return searchEngine.resolveAll(rawSuggestions, allowCategorySuggestions = false).mapValue {
-                resultFactory.createPlaceAutocompleteSuggestions(it)
-            }
-        }
-
-        val suggestions = rawSuggestions
+        val suggestions: List<Expected<Exception, PlaceAutocompleteSuggestion>> = rawSuggestions
             .mapNotNull { suggestion ->
-                val type = when (val suggestionType = suggestion.type) {
+                val type: PlaceAutocompleteType = when (val suggestionType = suggestion.type) {
```

Comment:
Add comments to describe what is happening here.
Note that we want category and brand results to be shown in their raw form (see just below in this function)

Reply:
This was just a type annotation, I reverted this unnecessary change.